### PR TITLE
🐛 A4A: allow-scripts for Safari in no-signing exp

### PR DESCRIPTION
**Pull request by calebcordry**
Partial for #27189

### CODE REVIEW

File: extensions/amp-a4a/0.1/secure-frame.js [hunk 1]
```diff
@@ -62,13 +63,19 @@ export const createSecureDocSkeleton = (url, sanitizedHeadElements, body) =>
...
  * @param {string} title
  * @param {string} height
  * @param {string} width
  * @return {!HTMLIFrameElement}
  */
-export function createSecureFrame(document, title, height, width) {
+export function createSecureFrame(win, title, height, width) {
+  let sandbox = sandboxVals.join(' ');
+  if (Services.platformFor(win).isSafari()) {
+    sandbox += ' allow-scripts';
```
Comment (powerivq) [inline extensions/amp-a4a/0.1/secure-frame.js:75]: might be cleaner doing `sandboxVals.push('allow-scripts')`

**Review by powerivq — COMMENTED** [review]
otherwise looks good

**Review by powerivq — COMMENTED** [review]
otherwise looks good

**Review by lannka — APPROVED** [review]

File: extensions/amp-a4a/0.1/secure-frame.js (hunk 1 above)
Comment (calebcordry) [reply extensions/amp-a4a/0.1/secure-frame.js:75]: This was my first instinct, I think that would keep adding to the array every time it is called?

Comment (powerivq) [reply extensions/amp-a4a/0.1/secure-frame.js:75]: Didn't notice it's a constant. How about

```
sandbox = sandboxVals.concat(['allow-scripts']);
```

Comment (powerivq) [reply extensions/amp-a4a/0.1/secure-frame.js:75]: Look like the only usage of `sandboxVal` is here. Why don't we set the whole space-separated sandbox attribute as a string constant?

**Review by powerivq — APPROVED** [review]

File: extensions/amp-a4a/0.1/secure-frame.js (hunk 1 above)
Comment (calebcordry) [reply extensions/amp-a4a/0.1/secure-frame.js]: changed to string.

**Pull request merged**